tests/codegen/snapshot/test_enum.py

```diff
@@ -40,12 +40,30 @@
           "errors": {
             "anyOf": [
               {
-                "type": "string",
-                "const": "err_first"
+                "type": "object",
```

Comment:
@blast-hardcheese I found that the river protocol allowed errors to have any type, including string literals. In order to make the test pass I had to change this test fixture. So to make this change would be imposing additional semantics that's not part of the protocol. Do we really want to?

Alternative could be to make `ErrorType` allow for `RiverUnknownValue` as an option.

Reply:
I added two tests that verify the functionality